crypto/rsa/internal.h

```diff
@@ -64,6 +64,7 @@
 extern "C" {
 #endif
 
+const uint32_t GFp_BN_BLINDING_COUNTER;
```

Comment:
We can drop this. We only put stuff in the headers if it is used by other C code in other files, not if it is used by only Rust code.